mkdocs.yml

```diff
@@ -163,72 +163,11 @@ markdown_extensions:
 # Plugins
 plugins:
   - search
+  - awesome-pages
 
 # Page Tree/Sidebar
 # https://www.mkdocs.org/user-guide/writing-your-docs/#configure-pages-and-navigation
+# https://3os.org/utilities/markdown-cheatsheet/awesome-pages/
+# We now use awesome-pages plugin, so check `.pages` files inside renovate docs folder
 nav:
-  - Home: 'index.md'
-  - Getting Started:
-      - 'Use Cases': 'getting-started/use-cases.md'
-      - 'Running Renovate': 'getting-started/running.md'
-      - 'Self-Hosting Examples': 'examples/self-hosting.md'
-      - 'Installing & Onboarding': 'getting-started/installing-onboarding.md'
-      - 'Private Packages': 'getting-started/private-packages.md'
-  - Troubleshooting: 'troubleshooting.md'
-  - Configuration:
-      - 'Self-hosted': 'self-hosted-configuration.md'
-      - 'Repository': 'configuration-options.md'
-      - 'Presets': 'config-presets.md'
-  - Key Concepts:
-      - 'Presets': 'key-concepts/presets.md'
-      - 'Dependency Dashboard': 'key-concepts/dashboard.md'
-      - 'Pull Requests': 'key-concepts/pull-requests.md'
-      - 'Renovate Scheduling': 'key-concepts/scheduling.md'
-      - 'Automerge': 'key-concepts/automerge.md'
-  - Renovate Modules:
-      - 'Platforms': 'modules/platform.md'
-      - 'Managers': 'modules/manager.md'
-      - 'Datasources': 'modules/datasource.md'
-      - 'Versioning': 'modules/versioning.md'
-  - Language Support:
-      - 'Bazel': 'bazel.md'
-      - 'Docker': 'docker.md'
-      - 'Golang': 'golang.md'
-      - 'Java': 'java.md'
-      - 'JavaScript': 'javascript.md'
-      - 'Node.js Versions': 'node.md'
-      - 'Nuget': 'nuget.md'
-      - 'PHP': 'php.md'
-      - 'Python': 'python.md'
-      - 'Ruby': 'ruby.md'
-  - Deep Dives:
-      - 'Should you Pin your Javascript Dependencies?': 'dependency-pinning.md'
-      - 'Config Template Editing': 'configuration-templates.md'
-      - 'Updating and Rebasing Branches': 'updating-rebasing.md'
-      - 'Semantic Commit Messages': 'semantic-commits.md'
-      - 'Noise Reduction': 'noise-reduction.md'
-  - Included Presets:
-      - 'Compatibility Presets': 'presets-compatibility.md'
-      - 'Default Presets': 'presets-default.md'
-      - 'Docker Presets': 'presets-docker.md'
-      - 'Full Config Presets': 'presets-config.md'
-      - 'Group Presets': 'presets-group.md'
-      - 'Helper Presets': 'presets-helpers.md'
-      - 'Monorepo Presets': 'presets-monorepo.md'
-      - 'npm Presets': 'presets-npm.md'
-      - 'Package Presets': 'presets-packages.md'
-      - 'Preview Presets': 'presets-preview.md'
-      - 'Regex Manager Presets': 'presets-regexManagers.md'
-      - 'Replacement Presets': 'presets-replacements.md'
-      - 'Schedule Presets': 'presets-schedule.md'
-      - 'Workaround Presets': 'presets-workarounds.md'
-  - All Other:
-      - User Stories:
-          - 'Swissquote': 'user-stories/swissquote.md'
```

Comment:
I'm running this in Codespaces now, and just clicking around on things. I think something broke the link in the sidebar to the Swissquote user story:

```bash
WARNING  -  [11:10:39] "GET /user-stories/swissquote.md HTTP/1.1" code 404
```

Steps to reproduce:

1. Open Codespace on this PR branch and let all the setup scripts finish
1. Run `make serve` and use your browser to preview the site
1. Go to the docs sidebar, expand _All other_ section -> expand _User Stories_ -> select the link to the Swissquote article
1. End up at a `404 - Not found` page

Reply:
yes, i need to make some more `.pages` files for subdirectories 😕